dataset/dictionary/artifacts.json5

```diff
@@ -2,133 +2,133 @@
   {
     en: "artifact",
     ja: "聖遺物",
-    zh: "圣遗物",
+    zhCN: "圣遗物",
```

Comment:
Oh, sorry, I meant I already did this change and you didn't have to do this change.
(But you don't have to revert this change.)

Reply:
So do I need to undo this operation? By replace all the zhCN to zh in all files.

Reply:
No, you don't have to undo that.